### PR TITLE
Remove spare commands from 'git-elegant'

### LINKED ISSUE #77: Remove `clear-local` and `feature` commands from main executable
Commands `clear-local` and `feature` were unintentionally added to main executable with pull request #70.

### PR DESCRIPTION
**Pull request by extsoft**
Resolves #77

### CODE REVIEW

**Comment by AntonKarabaza** [comment]
@extsoft Please, resolve conflicts.

**Comment by extsoft** [comment]
> @extsoft Please, resolve conflicts.
@AntonKarabaza Done.

File: src/main/git-elegant [hunk 1]
```diff
@@ -85,21 +85,9 @@ __batch() {
...
-clear-local() {
-    git-elegant pull $MASTER
-    local cmd="git branch -lvv | grep gone | awk {'print \$1'}"
-    __loop "git branch -d" $(eval "$cmd") || \
-        (
-            __gm "There are unmerged branches:" && \
-            __loop "__gm -" $(eval "$cmd") && \
-            __batch "Do you want to delete all unmerged branches?" "Delete this?" "git branch -D" $(eval "$cmd")
-        )
+rebase() {
```
Comment (AntonKarabaza) [inline src/main/git-elegant]: @extsoft `rebase` is also moved to separate executable, so we don't need it here.

Comment (extsoft) [reply src/main/git-elegant]: Done - https://github.com/extsoft/elegant-git/pull/78/commits/792a9972c302e8632676c15375d83bb9fe492403

**Comment by AntonKarabaza** [comment]
@extsoft Good.

**Comment by extsoft** [comment]
@rultor please merge

**Pull request merged**